packages/client/src/schema/repository.ts

```diff
@@ -2107,7 +2113,7 @@ export const initObject = <T>(
   }
 
   const record = { ...data };
-  const metadata =
+  const metadata: Object | undefined =
```

Comment:
Shouldn't be needed, right? (`Object` is normally something we avoid)

```suggestion
  const metadata =
```

Reply:
It appears to be necessary, without it I get: `error  Unexpected any value in conditional. An explicit comparison or type cast is required  @typescript-eslint/strict-boolean-expressions`